src/chrome/content/rules/Desjardins.xml

```diff
@@ -1,9 +1,14 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://accesd.desjardins.com/ => https://accesd.desjardins.com/: (28, 'Connection timed out after 10001 milliseconds')
+Fetch error: http://accesd.affaires.desjardins.com/ => https://accesd.affaires.desjardins.com/: (28, 'Connection timed out after 10000 milliseconds')
+
```

Comment:
HTTPS Everywhere definitely includes an active adversary in its threat model. We are specifically interested in preventing sslstrip-style attacks where a user navigates to a default HTTP page and gets stuck on HTTP by a MITM.

However, in this case, the admins of accessd.desjardins.com specifically don't expect users to ever manually navigate to their site, because their site would be broken in that situation. Clearly, by not providing port 80 service, they expect people to only reach their site through links that include the HTTPS scheme.

One issue I hadn't considered is that an adversary could modify a non-HTTPS page from which a user links to https://accessd.desjardins.com/, so it links to the HTTP version, and then intercept that HTTP version and insert a fake page. So maybe this should stay.

However, this implies that for every HTTPS site on the Internet, we should have a rule in HTTPS Everywhere for it. This will very quickly become untenable. I'd like to develop some framework for deciding when a rule is needed and when it is not.

In the meantime, how is this for a proposal:
 - If a site provides service on HTTPS, but times out or refuses connection on port 80, it can still have a rewrite rule, but it must be a simple one (http -> https, same hostname).
 - If a site does respond on port 80, but is broken in some way (never returns 200, goes into redirect loops, etc), it cannot have a rewrite rule because it's impossible to define whether the rewritten content is equivalent to the unrewritten content.